receiver/podmanreceiver/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/podman
 go 1.17
 
 require (
+	github.com/docker/docker v20.10.10+incompatible
```

Comment:
why is this required?

Reply:
We are importing the structure of events from the docker events as mentioned in [container/podman](https://github.com/containers/podman/blob/85bad0cc7c68b71ab7ddb6ed09b862145c6c6d0e/pkg/domain/entities/events.go#L13).

Reply:
Do you *have* to? We already define structs to parse other API responses so probably should do the same here